### ISSUE TITLE
Visibility of icons when the test is highlighted/selected

### ISSUE THREAD

**Issue by vladeck** [hbenl/vscode-test-explorer#90]
This is a nitpick from me, but when using Light+ theme in VSCode, visibility of icons for selected test is not good, as in screenshot:

![Annotation 2019-08-12 143428](https://user-images.githubusercontent.com/16443718/62865321-6dbff800-bd0e-11e9-872e-ff318392044e.png)

If they would become white (as does the text), then they would be much easier to view.

**Comment by hacknug** [comment]
I'd say they should use the same dark grey those icons on the right. UI-wise that's what the whole app does.

**Comment by hbenl** [comment]
It would be nice if the icons could change color like the text does. Unfortunately, I don't see how I could do this, the API doesn't allow me to specify different icons for the selected element. Looking at other extensions and even built-in icons in the VS Code UI, they all remain dark even when the background becomes dark because the element was selected.

**Comment by vladeck** [comment]
@hbenl I do not believe that this should have been closed so eagerly. This issue can be resolved, as demonstrated in the screenshot:

![Annotation 2019-09-02 174001](https://user-images.githubusercontent.com/16443718/64125009-da2b9580-cda8-11e9-9301-ffa0fb241fe7.png)

What you can observe in the screenshot is the white outline present for he light theme.

Here is the screenshot when using dark theme:

![Annotation 2019-09-02 174207](https://user-images.githubusercontent.com/16443718/64125083-0e06bb00-cda9-11e9-8fc5-676c36ae0e6f.png)

Also note that some of square-in-circle icon (icon in front of the specs) is not the color used by the rest of the icons, so it looks "pale" in both light and dark themes.

**Comment by hbenl** [comment]
@vladeck Thanks, the white outline is an excellent solution (as your screenshots show).

**Comment by numaru** [comment]
The vscode team is working on a way to [dynamically set the color of the icons](https://github.com/microsoft/vscode/issues/78889) and it is in the [iteration plan for september](https://github.com/microsoft/vscode/issues/80591). microsoft/vscode#78889 is marked related to microsoft/vscode#55857. Also, there is a good chance to see a new api in the next version of vscode to use theme-able icons.
